build.yaml

```diff
@@ -0,0 +1,19 @@
+targets:
+  $default:
+    builders:
+      json_serializable:
+        options:
+          any_map: false
+          checked: false
+          constructor: ""
+          create_factory: true
+          create_field_map: false
+          create_json_keys: false
+          create_per_field_to_json: false
+          create_to_json: true
+          disallow_unrecognized_keys: false
+          explicit_to_json: true
```

Comment:
@BLACKADDER-VIII @CarlosD77  this is very important, it's painful i fyou don't do this